hw/system/occamy/src/occamy_xilinx.sv.tpl

```diff
@@ -146,16 +148,6 @@ import ${name}_pkg::*;
     .pcie_cfg_rsp_i  ('0),
     // Tie the HBM interrupts to zero.
     .ext_irq_i ({8'b0, ext_irq_i}),
-    .apb_hbm_cfg_req_o (),
-    .apb_hbm_cfg_rsp_i ('0),
-    .hbi_cfg_req_o (),
-    .hbi_cfg_rsp_i ('0),
-    .apb_hbi_ctl_req_o (),
-    .apb_hbi_ctl_rsp_i ('0),
-    .hbm_phy_cfg_req_o (),
-    .hbm_phy_cfg_rsp_i ('0),
-    .hbm_seq_req_o (),
-    .hbm_seq_rsp_i ('0),
```

Comment:
Why remove these?

Reply:
These are not present in `occamy_top` anymore / got renamed

- `apb_hbm_cfg` -> `hbm_cfg`
- `hbi_cfg`, `apb_hbi_ctl`, 'hbm_phy_cfg_', 'hbm_seq' -> removed

Reply:
Could you do me a favour and update these while you're at it? :-)